app/models/contract.rb

```diff
@@ -86,9 +87,11 @@ def self.issued_by(issuer, *ids)
     where.has { name.op('COLLATE', sql(collate)).matches(pattern)}
   }
 
-  scope :by_account, lambda { |account| where({ :user_account_id => account.id } ) }
+  scope :by_account, ->(account) { where.has { user_account_id == account } }
```

Comment:
@hallelujah Because otherwise it does not work for the refactoring that you requested in https://github.com/3scale/porta/pull/555#discussion_r251474203
You can check it by running `spring m test/unit/account_test.rb:30` 😄